lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -407,14 +367,41 @@ def test_404_not_found_program(self):
             self.get_url(self.program_uuid, CourseKey.from_string("course-v1:fake+fake+fake")),
             self.get_url(self.program_uuid, self.course_not_in_program_key),
         ]
-        post_data = [self.learner_enrollment("A")]
+        request_data = [self.learner_enrollment("A")]
         for path_404 in paths:
-            response = self.client.post(path_404, post_data, format="json")
+            response = self.request(path_404, request_data)
             self.assertEqual(404, response.status_code)
 
+    def test_duplicate(self):
+        request_data = [
+            self.learner_enrollment("l1", "active"),
```

Comment:
I'm not in love with "l1" as a fake student key (I assume that's what it is).  Would prefer something like "learner-1".